### DISCUSSION TITLE
decrease datatable row height

### DISCUSSION THREAD

**Discussion by joao-arthur-moreira** [primefaces/community#1103] [PrimeFaces Templates] [answered]
How do I decrease the row height of the datatable component? I'm using primefaces 12 and the poseidon template

**Comment by buraksaglam089** [comment, marked as answer]
Hello,

In PrimeFaces 12 with the Poseidon theme, decreasing the DataTable row height requires custom CSS. There's no built-in property for this adjustment.  You can add CSS rules to decrease DataTable row's.

Regards.
